cmd/install/assets/hypershift_operator.go

```diff
@@ -110,6 +111,7 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 		fmt.Sprintf("--enable-ocp-cluster-monitoring=%t", o.EnableOCPClusterMonitoring),
 		fmt.Sprintf("--enable-ci-debug-output=%t", o.EnableCIDebugOutput),
 		fmt.Sprintf("--private-platform=%s", o.PrivatePlatform),
+		fmt.Sprintf("--expose-kube-admin-pwd=%v", o.ExposeKubeAdminPassword),
```

Comment:
this I believe ultimately we will want on the HostedCluster/HostedControlPlane CRD spec (likely somewhere in the oauth config section that already exists......

Reply:
We should add this parameter to the cluster create command not the install command:
https://github.com/openshift/hypershift/blob/718664c03cba6906d7aa7c5a51b406a35fe21ae0/cmd/cluster/cluster.go#L14

https://github.com/openshift/hypershift/blob/071e615acb15aad3eac83f9aeed108d47cfb3daf/api/fixtures/example.go#L258